src/persistence/entity/hash.user.entity.ts

```diff
@@ -0,0 +1,12 @@
+import { Entity, PrimaryGeneratedColumn, JoinColumn, OneToOne } from "typeorm";
+import { User } from "./user.entity";
+
+@Entity()
+export class HashUser {
+    @PrimaryGeneratedColumn()
```

Comment:
I think since we are using a relation database, that this table must have also and ID primary generated, and use a hash like bcrypt or something like that to generate the hash, not just use an auto increment character (that also is a bad security practice)